api/src/test/java/edu/wpi/first/shuffleboard/api/sources/recording/SerializationTest.java

```diff
@@ -61,12 +61,12 @@ public void testBooleanToBytes() {
   @Test
   @Disabled("Adapters will be moved")
```

Comment:
Looks like you've updated this test. Should it still be disabled?

Reply:
it was only updated because I changed the signature of a method it depends on.

Reply:
Does the test still fail?

Reply:
Ping!

Reply:
while I appreciate the thought, this is out of scope for this PR

Reply:
I mean if the test no longer fails it makes sense to re-enable it.